### ISSUE TITLE
HUGO resolving shortcode with /content and .md - gives 404

### ISSUE THREAD

**Issue by m4sterbunny** [OpenSecuritySummit/oss2018#129]
I am trying to find the correct syntax for the internal links
This page:
https://open-security-summit.org/faq/attendee-information/

Should link to this page:

https://open-security-summit.org/faq/sponsored_tickets/

Shortcode used is [funding]{{< ref "faq/sponsored_tickets" >}}

But that resolves as https://open-security-summit.org/content/faq/sponsored_tickets.md 404 error
the /content and the .md are doom
without those all works well- where am I mucking up??

**Comment by fzipi** [comment]
@m4sterbunny  You need to write it like this:
`[funding]({{< ref "faq/sponsored_tickets.md" >}})`

**Comment by m4sterbunny** [comment]
tried that syntax in the accommodation link below:
this  ({{< ref "faq/Accommodation" >}})
which creates this:
https://open-security-summit.org/oss2018/content/faq/Accommodation.md also a 404

**Comment by m4sterbunny** [comment]
for the sake of a / a page was lost

**Comment by fzipi** [comment]
@m4sterbunny Didn't undertood. What happened?

**Comment by m4sterbunny** [comment]
the syntax that worked was [funding]({{< ref "/faq/sponsored_tickets.md" >}})

I am freaked by my test-the-links experiences from yesterday. This am the attendee page works fine from my chrome window.... however my incognito window that I opened to avoid any caching issue influencing my results is still behaving badly. That first funding link works fine, then the accommodation link is a 404, and the final definition of done opens the attendee page (the page of origin). Gonna close that incognito and try a new. Yep new incognito works fine. 
I typically test links once I have updated a page, but I am not getting logical results within a few hours of updating the .md. Is there something I can do? clearing cache for chrome is a pest because I get logged out of everything- the incognito window idea was bust as the results were even more odd-ball.
